src/launcher/java/org/truffleruby/launcher/options/OptionsCatalog.java

```diff
@@ -611,6 +615,8 @@
                 return INTEROP_WRITE_CACHE;
             case "ruby.interop.invoke.cache":
                 return INTEROP_INVOKE_CACHE;
+            case "ruby.interop.new.cache":
+                return INTEROP_NEW_CACHE;
```

Comment:
It's not very nice for the user to have 26+ options about caches. In practice it means I have to resort to `grep` to find what I want in `jt ruby -Xoptions`.